Examples/ExamplesApp/Examples.swift

```diff
@@ -42,31 +42,43 @@ struct Examples: View {
 }
 
 extension ExampleList {
-    @available(macCatalyst, unavailable)
-    static let augmentedReality = Self(
-        name: "Augmented Reality",
-        examples: [
-            AnyExample("Flyover", content: FlyoverExampleView()),
-            AnyExample("Tabletop", content: TableTopExampleView()),
-            AnyExample("World Scale", content: WorldScaleExampleView())
-        ]
-    )
+    #if os(iOS) && !targetEnvironment(macCatalyst)
```

Comment:
Since line 56 starts at the beginning of this line, it should probably be the same here.

Reply:
Good catch! Will fix.